cdhweb/settings.py

```diff
@@ -234,7 +218,7 @@
 # URL that handles the media served from MEDIA_ROOT. Make sure to use a
 # trailing slash.
 # Examples: "http://media.lawrence.com/media/", "http://example.com/media/"
-MEDIA_URL = STATIC_URL + "media/"
```

Comment:
@rlskoeser here's where the media path changed.

Reply:
@thatbudakguy interesting, and that's the only change you had to make? So no need for any deploy or dev notes. We could probably remove the override from the project settings used for deploy after this, but no harm in having it there.